0027-managed-range-cache-range-lru.md

```diff
@@ -0,0 +1,58 @@
+---
+feature: managed_range_cache_range_lru
+authors:
+  - "jon-chuang"
+start_date: "2022/12/01"
+---
+
+# The Design of `ManagedRangeCache`'s Last-access-time Oracle - `RangeLru`
+
+## Summary
+The design of a mechanism for evicting ranges last accessed before a certain epoch for the `DynamicFilter` operator
```

Comment:
Does the "access" include both writes and reads?

Note that for DynamicFilter the writes are from outer side and the reads are from inner side (i.e. the 1-row side)

Reply:
After thinking for a while, I suppose that the "access" should only include reads from inner-side (moving the "row pointer" up or down), because the cache is designed used for reads only. The outer side is basically identical to a `MaterializeExecutor`

Reply:
Yes, access only indicates reads.